### PR TITLE
Add mixed images deprecation

### PR DESCRIPTION
**Pull request by ericlaw1979**

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -46,11 +47,13 @@ The web is constantly evolving to improve the user experience, security, and pri
 <!-- links -->  
 
 [DeployedEdgePoliciesSSLMinVersion]: /deployedge/microsoft-edge-policies#sslversionmin "SSLVersionMin - Microsoft Edge - Policies | Microsoft Docs"  
+[InsecureContentAllowedPolicy]:  /deployedge/microsoft-edge-policies#insecurecontentallowedforurls "InscureContentAllowedForUrls - Microsoft Edge - Policies | Microsoft Docs"
 
 [ChromePlatformStatus4664843055398912]: https://www.chromestatus.com/feature/4664843055398912 "Disallow sync XHR in page dismissal JavaScript | Chrome Platform Status"  
 [ChromePlatformStatus5088147346030592]: https://www.chromestatus.com/feature/5088147346030592 "Cookies default to SameSite=Lax | Chrome Platform Status"  
 [ChromePlatformStatus6251880185331712]: https://www.chromestatus.com/feature/6251880185331712 "Referrer Policy: Default to strict-origin-when-cross-origin | Chrome Platform Status"  
 [ChromePlatformStatus6246151319715840]: https://chromestatus.com/feature/6246151319715840 "Deprecate FTP support | Chrome Platform Status"
+[ChromePlatformStatus4926989725073408]: https://chromestatus.com/feature/4926989725073408 "Autoupgrade Image Mixed Contetn | Chrome Platform Status"
```
Comment (erik-anderson) [inline microsoft-edge/web-platform/site-impacting-changes.md:56]: Typo: "Contetn"

Comment (ericlaw1979) [reply microsoft-edge/web-platform/site-impacting-changes.md]: fixed thx

**Comment by jm-trd-ms** [comment]
Acrolinx score: 94

**Comment by ericlaw1979** [comment]
lgtm

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**